workbench.js

```diff
@@ -394,12 +447,16 @@ Workbench.prototype.waitForReceipt = function (txHash) {
   });
 };
 
+function checkForSandboxReceiptErrors(receipt, txHash) {
+  if (receipt.exception === 'out of gas') console.log('Out of gas: tx ' + txHash);
+}
 Workbench.prototype.waitForSandboxReceipt = function (txHash) {
   var self = this;
   return new Promise((resolve, reject) => {
     var called = false;
     function cb(err, receipt) {
       if (err) return reject(err);
+      checkForSandboxReceiptErrors(sandboxReceipt, txHash);
```

Comment:
I get an error here because `sandboxReceipt` is not defined. I think it should be `receipt`.